Backend/dev/migrations/dynamic-offer-driver-app/0491-adding-validated-using-DocumentVerificationConfig.sql

```diff
@@ -0,0 +1,2 @@
+update atlas_driver_offer_bpp.document_verification_config set validate_using = 'BACKEND' where document_type not in ('AadhaarCard', 'PanCard', 'ProfilePhoto') || merchant_id like (SELECT id from atlas_driver_offer_bpp.merchant where short_id = 'BRIDGE_CABS_PARTNER');
```

Comment:
Why Bridge ?

Reply:
because Bridge is not validated by frontend sdk